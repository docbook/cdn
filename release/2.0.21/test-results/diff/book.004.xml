<html xmlns:deltaxml="http://www.deltaxml.com/ns/well-formed-delta-v1" xmlns="http://www.w3.org/1999/xhtml" xmlns:dxx="http://www.deltaxml.com/ns/xml-namespaced-attribute" xmlns:dxa="http://www.deltaxml.com/ns/non-namespaced-attribute" deltaxml:deltaV2="A!=B" deltaxml:version="2.0" deltaxml:content-type="full-context">
   <head deltaxml:deltaV2="A=B">
      <link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css"/>
      <link rel="stylesheet" type="text/css" href="../../resources/css/default.css"/>
      <link href="../../resources/css/prism.css" rel="stylesheet" type="text/css"/>
      <meta charset="utf-8"/>
      <meta name="generator" content="DocBook XSL 2.0 Stylesheets Vxxx"/>
      <script type="text/javascript" src="../../resources/js/dbmodnizr.js"/>
      <title>Unit Test: book.004</title>
   </head>
   <body deltaxml:deltaV2="A!=B">
      <article deltaxml:deltaV2="A!=B" id="book" class="book">
         <header deltaxml:deltaV2="A=B" class="book-titlepage">
            <h1>Unit Test: book.004</h1>
         </header>
         <div deltaxml:deltaV2="A=B" class="lists-of-titles">
            <div class="toc">
               <p>
                  <b>Table of Contents</b>
               </p>
               <ul class="toc">
                  <li>
                     <span>
                        <a href="#chap1">1. First <code class="filename">Filename</code> Chapter</a>
                     </span>
                  </li>
                  <li>
                     <span>
                        <a href="#chap2">2. My Product<em>
                              <sup>plus</sup>
                           </em> description</a>
                     </span>
                  </li>
               </ul>
            </div>
         </div>
         <article deltaxml:deltaV2="A" id="chap1" class="chapter">
            <header class="chapter-titlepage">
               <h2>Chapter 1. First <code class="filename">Filename</code> Chapter</h2>
               <div class="abstract">
                  <p>bla in the first chapter</p>
               </div>
            </header>
            <p>blah with <code class="filename">filename</code>.</p>
         </article><div deltaxml:deltaV2="B" class="content">
            <article id="chap1" class="chapter">
               <header class="chapter-titlepage">
                  <h2>Chapter 1. First <code class="filename">Filename</code> Chapter</h2>
                  <div class="abstract">
                     <p>bla in the first chapter</p>
                  </div>
               </header>
               <div class="content">
                  <p>blah with <code class="filename">filename</code>.</p>
               </div>
            </article>
            <article id="chap2" class="chapter">
               <header class="chapter-titlepage">
                  <h2>Chapter 2. My Product<em>
                        <sup>plus</sup>
                     </em> description</h2>
               </header>
               <div class="content">
                  <p>blah.</p>
               </div>
            </article>
         </div><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><article deltaxml:deltaV2="A" id="chap2" class="chapter">
            <header class="chapter-titlepage">
               <h2>Chapter 2. My Product<em>
                     <sup>plus</sup>
                  </em> description</h2>
            </header>
            <p>blah.</p>
         </article>
      </article>
      <script deltaxml:deltaV2="A=B" src="../../resources/js/prism.js"/>
   </body>
</html>