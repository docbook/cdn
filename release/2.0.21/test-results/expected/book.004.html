<html xmlns="http://www.w3.org/1999/xhtml"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Unit Test: book.004</title><meta charset="utf-8" /><meta name="generator" content="DocBook XSL 2.0 Stylesheets V2.0.17" /><link rel="stylesheet" type="text/css" href="../../resources/css/default.css" /><script type="text/javascript" src="../../resources/js/dbmodnizr.js"></script><link href="../../resources/css/prism.css" rel="stylesheet" type="text/css" /><link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css" /></head><body><article id="book" class="book"><header class="book-titlepage"><h1>Unit Test: book.004</h1></header><div class="lists-of-titles"><div class="toc"><p><b>Table of Contents</b></p><ul class="toc"><li><span><a href="#chap1">1. First <code class="filename">Filename</code> Chapter</a></span></li><li><span><a href="#chap2">2. My Product<em><sup>plus</sup></em> description</a></span></li></ul></div></div><article id="chap1" class="chapter"><header class="chapter-titlepage"><h2>Chapter 1. First <code class="filename">Filename</code> Chapter</h2><div class="abstract"><p>bla in the first chapter</p></div></header><p>blah with <code class="filename">filename</code>.</p></article><article id="chap2" class="chapter"><header class="chapter-titlepage"><h2>Chapter 2. My Product<em><sup>plus</sup></em> description</h2></header><p>blah.</p></article></article><script src="../../resources/js/prism.js"></script></body></html>